com.varnamproject.Varnam.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
This should part of the upstream project as well

Reply:
But this desktop file is specific to the Flatpak packaged version only. I've seen other repos in Flathub also having this : https://github.com/flathub/org.geogebra.GeoGebra

Reply:
They're there because it's downstream packaging, without the projects' team involvement. The ideal situation is when the file is sent upstream and maintained there. It's not Flatpak/Flathub specific.